app/lib/removeColumns.js

```diff
@@ -35,10 +35,9 @@ function removeColumns() {
       const pomiDataReader = fs.createReadStream(`${OUTPUT_DIR}/${POMI_FILE}`);
       const reducedPomiDataWriter =
         fs.createWriteStream(`${OUTPUT_DIR}/${REDUCED_POMI_FILE}`);
-      const parser = parse({ from: 2 });
 
       pomiDataReader
-        .pipe(parser)
+        .pipe(parse())
```

Comment:
Done, kind of. I've pulled out `transformData` into a function so they are all now function calls.